### ISSUE TITLE
Sampling from four geographically divergent young female populations demonstrates forensic geolocation potential in microbiomes 

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#440]
Sampling from four geographically divergent young female populations demonstrates forensic geolocation potential in microbiomes – Thomas Clarke –Scientific Reports
https://www.nature.com/articles/s41598-022-21779-z

**Comment by MercyCherop** [comment]
Hello I'd like to be assigned to this. TIA!

**Comment by MercyCherop** [comment]
Hi @SvetlanaUP my work is ready to be reviewed. TIA!  
https://bugsigdb.org/Study_1172/Experiment_1

**Comment by SvetlanaUP** [comment]
Hi @MercyCherop, thank you for your contribution. Unfortunately this is a poor curation. Please do ask help in correcting your curation in Slack.

**CURATION RESULTS:**

| S/N | Criteria                                                           | Total Points | Points Earned |
|-----|--------------------------------------------------------------------|--------------|---------------|
| 1   | All elements marked "Needs review" (none "Incomplete")             | 1            | 1            |
| 2   | Correct study design                                                | 1            | 1             |
| 3   | Entered all relevant experiments and no irrelevant experiments      | 1            | 0             |
| 4   | Body site correctly identified (i.e. does not include multiple sites)| 1            | 1             |
| 5   | Condition entered according to contrast (correct EFO ontology)      | 1            | 0             |
| 6   | Contrast groups correctly identified                                | 1            | 0            |
| 7   | Groups correctly labeled as 1 and 0 (1=cases, 0=controls)           | 1            | 0             |
| 8   | Antibiotic exclusion correctly identified                           | 1            | 0             |
| 9   | Correctly identified sequencing details                             | 2            | 1             |
| 10  | Identified correct statistical test                                 | 1            | 0             |
| 11  | Identified MHT correction                                           | 1            | 0             |
| 12  | Correctly recorded matched on factors                               | 1            | 0             |
| 13  | Entered correct number of statistical tests per experiment          | 1            | 0             |
| 14  | All diversity measures identified                                   | 1            | 0             |
| 15  | Diversity results correctly entered as increased/decreased/unchanged| 1            | 0             |
| 16  | All signature sources correctly identified (-1 for each error)      | 2            | 0             |
| 17  | Abundance direction correctly selected                              | 1            | 0             |
| 18  | Members of signatures identified correctly                          | 2            | 0             |
| 19  | Correct use of NCBI taxonomy                                        | 2            | 0             |

**Total = 4/23**

**Comment by MercyCherop** [comment]
Alright will do! Thank you.

**Comment by SvetlanaUP** [comment]
@Cateline https://github.com/Cateline could you please team up with @MercyCherop and correct this curation. Thanks!

**Comment by Cateline** [comment]
> @Cateline https://github.com/Cateline could you please team up with @MercyCherop and correct this curation. Thanks!

Yes, let me link up with her on Slack

**Comment by MercyCherop** [comment]
Hi @Cateline let me know when you're available to discuss!

**Comment by SvetlanaUP** [comment]
Hello @MercyCherop @Cateline, just to check what's the status of this curation i.e. is it corrected?

**Comment by MercyCherop** [comment]
Hi @SvetlanaUP We hit a little bump in the road but we're still actively working in it!

**Comment by SvetlanaUP** [comment]
@MercyCherop ok, do not hesitate to ask for help on our Slack channel. There are curators supporting each other for different details and doubts in curating.

**Comment by KateRasheed** [comment]
Good afternoon @MercyCherop . Welldone on your curation. I see that some of your signatures have errors. If you enter the NCBI ID, it would correct the errors.

I would love to collaborate with you and @Cateline on the correction of this curation.

Thank you so much.

**Comment by Cateline** [comment]
Hi @KateRasheed , yes you can. We can organize a huddle and work through it together. The part we are having a problem with is the comparisons(Group 0 and 1), especially using Fig. 1 and 3(with their supplementary tables S6 and S7)

**Comment by KateRasheed** [comment]
@Cateline I just checked Fig. 1 and 3. They are actually not curatable. The curatable signatures are in the supplementary table. 

Let's discuss on slack.

**Comment by Cateline** [comment]
Okay. I'll schedule a meeting with you, @MercyCherop and I

**Comment by MercyCherop** [comment]
Hi @KateRasheed Looking forward to connecting!

**Comment by KateRasheed** [comment]
Good evening @SvetlanaUP . Corrections have been made to the study. 

Thank you so much.

@MercyCherop @Cateline

**Comment by SvetlanaUP** [comment]
@MercyCherop @Cateline @KateRasheed 
almost all corrections have been made, except the signatures: there’s at least one missing signature in almost all 8 experiments (observed by @Buraah https://github.com/Buraah)
Please do check all your signatures.

**Comment by Cateline** [comment]
Noted

**Comment by KateRasheed** [comment]
Good afternoon @SvetlanaUP @Buraah. Thank you so much the observation.

**Comment by KateRasheed** [comment]
I have added Parabacteroides merdae to signature 1 of experiment 3.
I didn't see any missing signature in experiment 4, 5, 7, 8, 9 and 10 (I didn't record taxa with 0.05 and above adjusted p-value)
I removed Odoribacter Splanchinus and Ruminococcus sp_5_1_39BFAA from signature 2 of experiment 6, because they are above the threshold.
I also corrected the signature error(streptococcus salivarius subsp. thermophilus) in signature 2 of experiment 9 with its current name(streptococcus thermophilus).

Thank you so much @SvetlanaUP @Buraah
